BZs mgmt-console-1 (#37113)
openshift/openshift-docs: release_notes/ocp-4-9-release-notes.adoc

[id="ocp-4-9-web-console-admin-perspective-bug-fixes"]
==== Web console (Administrator perspective)

* Previously, all rows on the `PF4` table were rerendering. With this update, the content in `React.memo` was wrapped so the content does not rerender on every scroll event.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=1856355[*BZ#1856355*])

> * Previously, the charts in *Cluster Utilization* in the {product-title} web console displayed the data time span in a confusing manner. For example, if the six hour time span option is selected, but data exists only for the final three hours, those three data points are stretched to fill the entire chart. The first three hours are not displayed. This could result in the assumption that the chart is showing the full six-hour time span. To avoid confusion, the charts now show blank space for missing information. In this example, the chart displays the entire six-hour time span with data starting at the fourth hour. The first three hours are blank.

Reviewer: ```suggestion
* Previously, the charts in *Cluster Utilization* in the {product-title} web console displayed the data time span in a confusing manner. For example, if the six-hour time span option is selected, but data exists only for the final three hours, those three data points are stretched to fill the entire chart. The first three hours are not displayed. This could result in the assumption that the chart is showing the full six-hour time span. To avoid confusion, the charts now show blank space for missing information. In this example, the chart displays the entire six-hour time span with data starting at the fourth hour. The first three hours are blank.
```
(Matches consistency with later instances in this paragraph.)

(link:https://bugzilla.redhat.com/show_bug.cgi?id=1904155[*BZ#1904155*])

* Previously, `NetworkPolicy` was not translated to Korean or Chinese on the web console. With this update, `NetworkPolicy` is now translated correctly when viewing the web console in Korean or Chinese.

> (link:https://bugzilla.redhat.com/show_bug.cgi?id=1965930[*BZ#1965930])

Author: missed *, will fix

* Previously, an issue with the `Needs Attention` state of the *Console Overview* section showed Operators as `upgrading`, even if they were not upgrading. This update fixes the `Needs Attention` state so that the correct status of an Operator is displayed.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=1967047[*BZ#1967047*])

> * Previously, the alert for a failed Cluster Service Version (CSV) displayed a generic `status.message` that did not help troubleshoot the the failed CSV. With this update, the behavior for copied CSVs show a helpful message and a link to the original CSV to troubleshoot.

Reviewer: This seems like it should be `s/show/shows`:
> the behavior for copied CSVs show a helpful message

But it might read better by dropping "the behavior for" and writing just, "With this update, copied CSVs show a helpful message and a link to the original CSV to troubleshoot." 🤷

Author: I like it :)

(link:https://bugzilla.redhat.com/show_bug.cgi?id=1967658[*BZ#1967658*])

* Previously, a user was unable to use the drop-down options in the masthead with a keyboard. With this update, users are now able to access the drop-down options with a keyboard.

> (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967658[*BZ#1967658*])

Reviewer: This is a repeat of the previous BZ ID. 
```suggestion
(link:https://bugzilla.redhat.com/show_bug.cgi?id=1967979[*BZ#1967979*])
```

* Previously, a utility function used for matching Operator-owned resources with their owners would return false matches. Consequently, *Managed by* links on Operator-owned resource pages would sometimes link to the incorrect URL. This fix updates the function logic to correctly match owned Operators. As a result, *Managed by* links now link to the correct URL.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=1970011[*BZ#1970011*])

> * Previously, the web console *OperatorHub* leads users to unrelated install plans. With this update, the *OperatorHub* links users to the Operator details *Subscription* tab to view installation progress.

Reviewer: ```suggestion
* Previously, the *OperatorHub* web console interface would lead users to unrelated install plans. With this update, *OperatorHub* links users to the Operator *Subscription* details tab to view installation progress.
```

> (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970466[*BZ#1970466])

Author: missed *, will fix
